indigo/distribution.yaml

```diff
@@ -2495,6 +2495,21 @@ repositories:
       url: https://github.com/Hacks4ROS/h4r_x52_joyext.git
       version: develop
     status: maintained
+  haf_grasping:
+    doc:
+      type: git
+      url: https://github.com/davidfischinger/haf_grasping.git
+      version: master
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/davidfischinger/haf_grasping/tree/v0.5-beta
```

Comment:
Thank you.
Regards,
David

2015-07-28 18:52 GMT+02:00 Dirk Thomas <notifications@github.com>:

> In indigo/distribution.yaml
> <https://github.com/ros/rosdistro/pull/9002#discussion_r35670834>:
>
> > @@ -2495,6 +2495,21 @@ repositories:
> >        url: https://github.com/Hacks4ROS/h4r_x52_joyext.git
> >        version: develop
> >      status: maintained
> > +  haf_grasping:
> > +    doc:
> > +      type: git
> > +      url: https://github.com/davidfischinger/haf_grasping.git
> > +      version: master
> > +    release:
> > +      tags:
> > +        release: release/indigo/{package}/{version}
> > +      url: https://github.com/davidfischinger/haf_grasping/tree/v0.5-beta
>
> This is not a proper release repository. Please follow the bloom tutorials
> (http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) to release your
> package or remove the release entry and only keep doc and source.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros/rosdistro/pull/9002/files#r35670834>.
>



-- 
Dr. David Fischinger
Vienna University of Technology
Automation & Control Institute
Gusshausstr. 27/376
1040 Vienna, Austria, Europe
Phone: +43 158801 376622
fischinger@acin.tuwien.ac.at